### PR TITLE
Support custom domain when using connectors and console

### PR DESCRIPTION
**Pull request by Lavaburn**
Replace hard-coded "cluster.local" by clusterDomain value

### CODE REVIEW

File: charts/redpanda/templates/console/configmap-and-deployment.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-{{/*
```
Comment (alejandroEsc) [inline charts/redpanda/templates/console/configmap-and-deployment.yaml:1]: was this removed accident? If not then we need to do the same on the other end of the boilerplate

**Review by alejandroEsc — CHANGES_REQUESTED** [review]
thank you for the contribution, this is looking great! just one ask

File: charts/redpanda/templates/console/configmap-and-deployment.yaml (hunk 1 above)
Comment (Lavaburn) [reply charts/redpanda/templates/console/configmap-and-deployment.yaml]: Sorry, that was accidental. I have fixed the commit now.

**Review by alejandroEsc — APPROVED** [review]
lgtm, thank you again!

**Comment by alejandroEsc** [comment]
Updating branch, will merge once I am given the go ahead, there is another PR we are waiting for first,

**Review by RafalKorepta — APPROVED** [review]

**Pull request merged**